### ISSUE TITLE
rsync Deployments MVP

### ISSUE THREAD

**Issue by bmackinney** [pantheon-systems/documentation#874]
rsync deploy as content, inplace of existing git method via site dashboard.

- [x] draft new deploy script using rsync for the docs Multidev env
- [x] draft new deploy script using rsync for the live env
- [x] include backups on app server
- [x] strategize revert options
- [x] test on business site (static docs)

**Comment by rachelwhitton** [comment]
@bmackinney can you provide a list of essential scenarios I should focus on for behat tests? Here's what I have so far: 

Current Scenarios: 
-  Scenario: The homepage text includes expected content, such as links to all articles, etc.
-  Scenario: Verify Contributor CTA button and links exists

I pushed branch [874-behat-tests](https://github.com/pantheon-systems/documentation/tree/874-behat-tests) which includes a javascript scenario. Currently uses default browser (FireFox) of driver. Using FF because for some reason a few feature context methods aren't working in Chrome (e.g. press and click for testing contributor button). I can add more info about replicating the Chrome issues if you're interested. Otherwise, I'll keep using FireFox.

**Comment by ari-gold** [comment]
@rachelwhitton Btw. not sure if this is the case, but if the Chrome issue is only happening during Behat tests and not when using Chrome as usual, then you might make sure you're using the latest Selenium Server.

**Comment by rachelwhitton** [comment]
Thanks @ari-gold I'll look into updating Selenium!

Here's more about the problem with Chrome - I'll check versions tomorrow for Selenium.

behat version 3.0.15

Adding `browser: chrome` to `selenium2:` in `behat.yml` causes test to fail:
```
    And I press "Contribute to Docs"  # FeatureContext::pressButton()
      Button with id|name|title|alt|value "Contribute to Docs" not found.

2 scenarios (1 passed, 1 failed)
18 steps (14 passed, 3 skipped, 1 failed)
```

I wrote up a [FeatureContext method](https://github.com/pantheon-systems/documentation/blob/874-behat-tests/features/bootstrap/FeatureContext.php#L22-L37) to check if a provided element ID exists. Replaced the above step with `Then I find the element with ID "cta-contrib-btn`and re-run the test. This time, the element is found and step is passed, but remaining tests fail because the element was not toggled onclick. 

I tried to toggle the dropdown button by adding `$elemendID->click();` to the [FeatureContext method](https://github.com/pantheon-systems/documentation/blob/874-behat-tests/features/bootstrap/FeatureContext.php#L22-L37) but that gives the following fatal error:
```
PHP Fatal error:  Call to a member function click() on a non-object in /Users/Rachelwhitton/sites/documentation/features/bootstrap/FeatureContext.php on line 36
```
